Assets/Scripts/Room.cs

```diff
@@ -19,9 +19,30 @@ public class Room : MonoBehaviour
 
     public UnityEvent<string> roomEnteredEvent = new UnityEvent<string>();
 
+    public string roomtype = "normal";
+
+    private int enemy_count;
+
+    private GameObject[] enemylist;
+
     // Start is called before the first frame update
     void Start()
-    {
+    {   
+        if (roomtype.Equals("normal")){
+            enemy_count = (int) Random.Range(2,6);
+            enemylist = Resources.LoadAll<GameObject>("Prefabs/Enemy");
+
+            for (int i=0; i<enemy_count; i++){
+                Debug.Log("Creating enemy number: " + i);
+                int index = Random.Range(0,enemylist.Length);
+                GameObject enemy = Instantiate(enemylist[index]);
+                enemy.SetActive(false);
+                enemy.transform.position = new Vector3(transform.position.x + Random.Range(-10f,10f),transform.position.y + Random.Range(-4f,4f),12.07283f);
+                enemies.Add(enemy);
```

Comment:
looks like some rooms listed as "normal" in the unity mainscene start with non-empty `enemies` lists.  Did you intend to append to enemies or generate the list from scratch in this code snippet?

Reply:
I thought I've deleted the lists. Let my check.